resources/datadog-integrations/project.rb.erb

```diff
@@ -43,8 +43,11 @@ end
 dependency 'preparation'  # creates required build directories
 dependency 'dd-check-<%= name %>-software'
 
+# Checks should require the datadog agent
+runtime_dependency 'datadog-agent'
```

Comment:
Yes, I believe so. I looked at how it's defined in the packager and tested it on both systems. We should exhaustively test this before we release. The tests do pass, so it does install fine.